### ISSUE TITLE
Angular 6

### ISSUE THREAD

**Issue by jdart** [actra-development/actra-ng-i18n-loader#2]
I'm looking at using this with an angular 6 app, will it work there? I don't have any webpack configuration to hook into, how would I wire this up?

**Comment by actra-gschuster** [comment]
Version 2.0.0 works with angular 6 - at least when you "hack" their config to add it to the loaders.

The angular team is working on a devkit-builder that allows specifiying custom webpack configs:
https://github.com/angular/angular-cli/tree/master/packages/angular_devkit/build_webpack

**Comment by digismack** [comment]
With Angular 5.x I had an ejected build with a modified webpack config. It was a pain to manage, but it worked. In Angular 6, ejecting has been temporarily disabled because of the change to the angular.json format.

Can you provide some insight on how to get the ng-i18n-aot-loader working in an Angular 6 project?

You mention above that you can "hack" their config to add it to the loaders. Can you provide some insight on how this can be accomplished?

**Comment by actra-gschuster** [comment]
I forked the ng-devkit package so I could modify build_angular, afterwards building it within a custom namespace and changing angular.json to use this new builder.
As far as I know the angular team is working on an additional builder (build_webpack -> https://github.com/angular/angular-cli/tree/master/packages/angular_devkit/build_webpack) that should allow to specify custom configs as with ng 5.

**Comment by msyesyan** [comment]
@actra-gschuster hi, I have created an angular demo project for use this loader and custom-webpack-loader, it only works when I set aot: false in angular.json, can you help me have a look? thanks very much . https://github.com/msyesyan/angular-custom-webpack-aot-demo

**Comment by actra-gschuster** [comment]
Haven't tested it against 7 or 8 yet, so for the moment I don't have a clue.
Do you get any error messages when AOT is enabled?

**Comment by msyesyan** [comment]
@actra-gschuster there is no any error messages, just have no effect, I still haven't found any clues for now

**Comment by actra-gschuster** [comment]
@msyesyan Is the demo repo you linked working in JIT?
I'll see if I find some time next weekend to check it out and get it working in AOT.

**Comment by msyesyan** [comment]
@actra-gschuster yes, the demo worked for jit, when change the `aot: true` in angular.json, then it won't work
